index.html

```diff
@@ -5,15 +5,19 @@
   <meta charset="UTF-8" />
   <meta name="description"
     content="Zakład Pogrzebowy w Pasłęku pomaga od blisko 30 lat w organizacji ceremonii pogrzebowych." />
+  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />
+
   <title>Zakład Pogrzebowy Pasłęk | Janina Cieśla</title>
   <link rel="stylesheet" href="style.css" />
   <link rel="preconnect" href="https://fonts.gstatic.com" />
-  <link href="https://fonts.googleapis.com/css2?family=Cormorant+Upright&family=Montserrat:wght@400;600&display=swap"
-    rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css2?family=Old+Standard+TT:ital,wght@0,400;0,700;1,400&display=swap"
```

Comment:
It's better to load fonts before your custom css. :) Actually, your custom css should appear as last file.

Reply:
> On mobiles I can see 'x' mark and hamburger icon appear at the same time, I also cannot use hamburger icon yet, should I?

Could you try use hamburger again? It should work.

Reply:
It works like gold 🚀  🎉

Reply:
❤️